deps/oob-1.0.2/jsonutil.c

```diff
@@ -217,6 +221,8 @@ char *prepareOOBevent(oobOcspData *ocspevent)
   vals = cJSON_CreateObject();
   cJSON_AddItemToObject(list, "Value", vals);
 
+  cJSON_AddItemToObject(vals, "DSN", dsn);
```

Comment:
What is dsn is NULL ? Wouldn't this crash ? Do you have a test for this ? cc @sfc-gh-azhan